Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Runs on the default latest node LTS image
+# Build stage
+FROM node:lts AS build
+
+## Create the solid community server directory
+RUN mkdir /community-server
+
+## Set current working directory
+WORKDIR /community-server
+
+## Copy the dockerfile's context's community server files
+COPY . .
+
+## Verify if there are known vulnerabilities in the dependencies
+RUN npm audit --production --audit-level=high
```

Comment:
Will this error and stop the build if it finds known vulnerabilities? Do you want it to?

Reply:
Yes and yes.